src/containers/SubjectPage/components/SubjectEditorChoices.tsx

```diff
@@ -67,16 +76,4 @@ const SubjectEditorChoices = ({
   );
 };
 
-SubjectEditorChoices.propTypes = {
-  editorsChoices: PropTypes.arrayOf(GraphQLResourceShape),
-  narrowScreen: PropTypes.bool,
-  wideScreen: PropTypes.bool,
-  locale: PropTypes.string.isRequired,
-};
-
-SubjectEditorChoices.defaultProps = {
-  narrowScreen: false,
-  wideScreen: false,
-};
```

Comment:
`DefaultProps` brukes til å sette default variabler på props, ser at du sletta disse men satt de ikke tilbake oppe, gjør det slik at funksjonaliteten er så likt som mulig.

Reply:
Vil det være nødvendig her da? Ved check av variablene så vil de uansett gi `false` med mindre de faktisk er `true`?

Reply:
Det stemmer, men jeg tenker at nå som vi har typescript er det greit å la den gjøre jobben sin og passe på at vi sender med alle props osv. Den optional proppen på narrow- og wide-screen føles veldig lazy, og er gjort kun for at vi skal slippe å sende disse propsa med i komponenten. Litt unødvendig å ha 3 states `true | false | undefined` hvor false og undefined brukes interchangeably (med mindre undefined skulle eksplisitt brukes til noe annet). Tenker at det er best å fjerne optional på disse og kun bruke `true | false` states, og samtidig la ts gjøre jobben sin ved å pirke på oss hvis vi glemmer å sende med alle props